src/modules/examples/hellohook.c

```diff
@@ -44,7 +45,7 @@ void clientChangeCallback(RedisModuleCtx *ctx, RedisModuleEvent e, uint64_t sub,
     REDISMODULE_NOT_USED(e);
 
     RedisModuleClientInfo *ci = data;
-    printf("Client %s event for client #%llu %s:%d\n",
+    printf("Client %s event for client #%"PRIu64" %s:%d\n",
```

Comment:
i have a vague memory of a PR that wanted to do a similar thing in redis INFO, and Salvatore arguing that it may not be portable and that we should use casting instead.
i can't locate that and i might be confusing it with something else..
FYI.

Reply:
AFAIK it's C99

Reply:
"About @madolson PR, because the macros could be missing in some systems, we never used them in the past. I'll just do what I did so far: casting to an equivalent type that is always well-defined: this is basically what @guybe7 did in the fix plus the casting part that is missing. Thanks both."

Reply:
`<inttypes.h>` is indeed C99: https://en.wikibooks.org/wiki/C_Programming/inttypes.h

Reply:
I suppose Salvatore rejected it simply because it wasn't used before in redis and wanted to keep the code consistent.
In my eyes it does make the code harder to read (due to macros and string concatenation, and also i'm guessing i'm a C89 guy).
But i don't mind that much either way.